src/MLJ.jl

```diff
@@ -46,7 +46,7 @@ end
 function predict(::EvoTreeClassifier, fitresult, Xnew)
     Xnew = MLJModelInterface.matrix(Xnew)
     pred = predict(fitresult, Xnew)
-    return MLJModelInterface.UnivariateFinite(fitresult.levels, pred)
+    return MLJModelInterface.UnivariateFinite(fitresult.levels, pred, pool=missing)
```

Comment:
This does not seem right. I would be happy to look into this but can you please just merge #69 and make this a separate exercise. The compat issue is holding things up at our end. We want to release a new MLJModels/MLJ but it's tricky if your package is not up-to-date (because of way model registry is updated.

Thanks.

Reply:
To be clear: there is no need for you to buy into new data front-end to use MLJModelInterface 0.4

Reply:
Also in the meantime, you may want to look at how the pool business is taken care of for XGBoost.jl:

https://github.com/alan-turing-institute/MLJXGBoostInterface.jl/blob/325c22d3875d9b84540138817660aeb9b7f842fc/src/MLJXGBoostInterface.jl#L589

https://github.com/alan-turing-institute/MLJXGBoostInterface.jl/blob/325c22d3875d9b84540138817660aeb9b7f842fc/src/MLJXGBoostInterface.jl#L622

https://github.com/alan-turing-institute/MLJXGBoostInterface.jl/blob/325c22d3875d9b84540138817660aeb9b7f842fc/src/MLJXGBoostInterface.jl#L643

https://github.com/alan-turing-institute/MLJXGBoostInterface.jl/blob/325c22d3875d9b84540138817660aeb9b7f842fc/src/MLJXGBoostInterface.jl#L660